cmd/kubernetes/webhook/main.go

```diff
@@ -62,8 +61,6 @@ func main() {
 	webhook.CreateWebhookResources(ctx)
 	webhook.SetTypes("kubernetes")
 
-	go gracefulTermination(ctx)
```

Comment:
Removing this won't impact on 4.13 or 4.12? asking because may be the gracefulTermination added for a reason :thinking:

Reply:
@savitaashture It will not impact in any version, anyhow on the webhook pod startup deletes the existing webhook installersets and creates new set.